### PR TITLE
Item & Container sheet V2 skeleton & header matter

### PR DESCRIPTION
**Pull request by Fyorl**
This PR contains the Item sheet V2 skeleton structure, including the header matter and associated styling.

### CODE REVIEW

File: module/applications/fields.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Create a text input for a StringField.
+ * @param {StringField} field               The field.
+ * @param {FormInputConfig<string>} config  The input configuration.
+ * @returns {HTMLElement|HTMLCollection}
+ */
+export function createTextInput(field, config) {
```
Comment (Fyorl) [inline module/applications/fields.mjs:7]: Basically only using this to polyfill the core `createTextInput` and allow attaching a class to the input.

File: templates/items/item-sheet-2.hbs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+        </div>
+
+        {{!-- Weight, Price, Quantity, Level, & School --}}
+        <div class="right common-fields {{#if isPhysical}}physical{{/if}}">
+
+            {{#if isPhysical}}
+
+            {{!-- Quantity --}}
+            <div class="quantity">
```
Comment (arbron) [inline templates/items/item-sheet-2.hbs:51]: Having the numbers on the left for weight & cost but on the right for quantity feels a bit off. I wonder if we can find an icon for quantity and use that instead:

Screenshot 2024-08-02 at 13 53 39

```html
<i class="fa-solid fa-cubes-stacked"></i> // for example
```

**Review by arbron — APPROVED** [review]
Noticed that minimized sheets are broken, but that might be in a future PR:

Screenshot 2024-08-02 at 13 54 15

File: templates/items/item-sheet-2.hbs (hunk 2 above)
Comment (Fyorl) [reply templates/items/item-sheet-2.hbs:52]: I see what you mean, but the quantity icon isn't quite cutting it for me. I will try some things out on the forward branch.

**Comment by Fyorl** [comment]
> Noticed that minimized sheets are broken, but that might be in a future PR:
> 
> Screenshot 2024-08-02 at 13 54 15

Not in a future PR but I will fix it on the forward branch.

**Pull request merged**